cmd/ibc/src/lib.rs

```diff
@@ -324,6 +351,49 @@ impl<'a> IbcHandler<'a> {
             }
         }
 
+        // The STATUS_MFR depends on the value of STATUS_WORD.  See
+        // Application Note 326 for details.
+        let mfr = [
+            (
+                0x0001u16,
+                &[
+                    (0u8, "BOOT_EVENT"),
+                    (1, "INPUT_LOW_EVENT"),
+                    (2, "CANCEL_EVENT"),
+                    (3, "ERASE_EVENT"),
+                    (4, "CLR_EVENT"),
+                    (5, "ERASE_OVFL_EVENT"),
+                    (6, "TIME_ERASE_EVENT"),
+                    (7, "Brown out detected (?)"),
+                ] as &[(u8, _)],
+            ),
+            (
+                0x0040,
+                &[
+                    (0, "Unit is off due to internal configuration"),
+                    (1, "Unit is off due to primary remote control"),
+                    (2, "Unit is off due to seconday remote control"),
+                    (3, "Unit is off due to PMBus command"),
+                    (4, "Unit is on due to internal configuration"),
+                    (5, "Unit is on due to primary remote control"),
+                    (6, "Unit is on due to seconday remote control"),
+                    (7, "Unit is on due to PMBus command"),
+                ],
+            ),
+            (0x0800, &[(0, "Power not good"), (1, "Power is good")]),
+        ];
+
+        // Always print the actual raw bits, even if they are 0.
+        println!("  {: <18}  {:#04x}", "STATUS_MFR", e.status_mfr);
+
+        if let Some(&(_, bits)) = mfr.iter().find(|&&(v, _)| v == status_word) {
+            for (value, meaning) in bits {
+                if e.status_mfr == *value {
+                    println!("    {value:#04x}: {meaning}");
+                }
+            }
```

Comment:
Printing unknown values would be good, something like (untested)
```suggestion
            if let Some((value, meaning)) = bits.iter()
                .find(|(value, _)| e.status_mfr == *value)
            {
                println!("    {value:#04x}: {meaning}");
            } else {
                println!("    {value:#04x}: Unknown meaning!");
            }
```